ledger/src/bank_forks_utils.rs

```diff
@@ -43,12 +70,16 @@ pub fn load(
             )
             .expect("Load from snapshot failed");
 
-            return blockstore_processor::process_blockstore_from_root(
-                genesis_config,
-                blockstore,
-                Arc::new(deserialized_bank),
-                &process_options,
-                &VerifyRecyclers::default(),
+            let snapshot_hash = (deserialized_bank.slot(), deserialized_bank.hash());
```

Comment:
@sakridge - this line might need to be changed if a different snapshot hash is used in the future